src/module/combat/combat.js

```diff
@@ -481,6 +481,7 @@ export class CombatSFRPG extends Combat {
         const chatData = {
             type: CONST.CHAT_MESSAGE_TYPES.OTHER,
             speaker: ChatMessage.getSpeaker({ actor: eventData.newCombatant, token: eventData.newCombatant?.token, alias: speakerName }),
+            whisper: eventData.newCombatant.hidden ? [game.user.data._id] : [],            
```

Comment:
This should use `ChatMessage.getWhisperRecipients("GM")` instead of using `[game.user.data._id]`.

Reply:
Awesome thanks! Done!